NitroxClient/GameLogic/Entities.cs

```diff
@@ -95,6 +97,22 @@ public void BroadcastEntitySpawnedByClient(WorldEntity entity)
             packetSender.Send(new EntitySpawnedByClient(entity));
         }
 
+        public IEnumerator SpawnNewEntities()
```

Comment:
Would make this private as it should only be called by `EnqueueEntitiesToSpawn()`, shouldn't it?